src/Altinn.Platform/Altinn.Platform.Storage/IntegrationTest/InstanceMultipartTests.cs

```diff
@@ -0,0 +1,180 @@
+        {
+            this.fixture = fixture;
+            this.client = this.fixture.Client;
+            this.storageClient = new InstanceClient(this.client);
+
+            CreateTestApplication();
+        }
+
+        /// <summary>
+        /// Make sure repository is cleaned after the tests is run.
+        /// </summary>
+        public void Dispose()
+        {
+            string requestUri = $"{versionPrefix}/instances?org={testOrg}";
+
+            HttpResponseMessage response = client.GetAsync(requestUri).Result;
+            string content = response.Content.ReadAsStringAsync().Result;
+
+            if (response.StatusCode == HttpStatusCode.OK)
+            {
+                JObject jsonObject = JObject.Parse(content);
+                List<Instance> instances = jsonObject["instances"].ToObject<List<Instance>>();
+                
+                foreach (Instance instance in instances)
+                {
+                    string url = $"{versionPrefix}/instances/{instance.Id}";
+
+                    if (instance.Data != null)
+                    {
+                        foreach (DataElement element in instance.Data)
+                        {
+                            string filename = element.StorageUrl;
+                            string dataUrl = "/data/" + element.Id;
+
+                            string dataDeleteUrl = url + dataUrl;
+
+                            client.DeleteAsync(dataDeleteUrl);
+                        }
+                    }
+
+                    string instanceUrl = $"{versionPrefix}/instances/{instance.Id}?hard=true";
+                    client.DeleteAsync(instanceUrl);
+                }
+            }
+
+            DeleteApplicationMetadata();
+        }
+
+        private Application CreateTestApplication()
+        {
+            ApplicationClient appClient = new ApplicationClient(client);
+
+            try
+            {
+                Application existingApp = appClient.GetApplication(testAppId);
+                return existingApp;
+            }
+            catch (Exception)
+            {
+                // do nothing.
+            }
+
+            LanguageString title = new LanguageString
+            {
+                { "nb", "Testapplikasjon" },
+                { "en", "Test application" }
+            };
+
+            return appClient.CreateApplication(testAppId, title);
+        }
+
+        private Application DeleteApplicationMetadata()
+        {
+            ApplicationClient appClient = new ApplicationClient(client);
+
+            Application existingApp = appClient.DeleteApplication(testAppId);
+
+            return existingApp;
+        }
+
+        /// <summary>
+        /// Store a multipart file in one post operation
+        /// </summary>
+        [Fact]
+        public async void StoreMultiPartFileInOnePostOperation()
+        {
+            Instance instance = new Instance()
+            {
+                Id = Guid.NewGuid().ToString(),
```

Comment:
Why do you create a guid her? The guid is generated by the storage service.

Reply:
My fault, removed now.